system/Language/en/Errors.php

```diff
@@ -11,6 +11,8 @@
 
 // Errors language settings
 return [
-    'whoops'     => 'Whoops!',
-    'weHitASnag' => 'We seem to have hit a snag. Please try again later...',
+    '404pageNotFound'    => '404 - Page Not Found',
+    '404sorryCannotFind' => 'Sorry! Cannot seem to find the page you were looking for.',
```

Comment:
These should be camel case. I know the caps next to the number looks weird but that's the protocol. My recommendation would be to drop the number, or if you really want to keep it then move it to the end. 
```suggestion
    'pageNotFound'    => '404 - Page Not Found',
    'sorryCannotFind' => 'Sorry! Cannot seem to find the page you were looking for.',
```